packages/mobile/src/utils/reachability.ts

```diff
@@ -0,0 +1,81 @@
+import { reachabilityActions } from '@audius/common'
+import type { NetInfoState } from '@react-native-community/netinfo'
+import NetInfo from '@react-native-community/netinfo'
+import { debounce } from 'lodash'
+import { AppState } from 'react-native'
+
+import { env } from 'app/services/env'
+import { dispatch } from 'app/store'
+
+const REACHABILITY_URL = env.REACHABILITY_URL
```

Comment:
In my testing this was available, and it looks like we're just grabbing it from the `.env` files where it's already populated:

```
// .env.dev; .env.stage
REACHABILITY_URL=https://staging.audius.co/204
```
```
// .env.prod
REACHABILITY_URL=https://audius.co/204
```